packages/flame_network_assets/example/pubspec.yaml

```
@@ -0,0 +1,18 @@
+name: example
```

```suggestion
name: flame_network_assets_example
```
This would collide with any other package named example.